bigtable/client/testing/embedded_server_test_fixture.h

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 Google Inc.
```

Comment:
s/Inc./LLC/ we changed this boilerplate recently.

Reply:
fixed